src/afl-fuzz-init.c

```diff
@@ -697,11 +713,9 @@ void read_testcases(afl_state_t *afl) {
 
     if (st.st_size > MAX_FILE) {
 
-      WARNF("Test case '%s' is too big (%s, limit is %s), skipping", fn2,
+      WARNF("Test case '%s' is too big (%s, limit is %s), partial reading", fn2,
```

Comment:
Did we ever check if we have issues with sharedmap input, mutations, splicing, ... when testcases are `MAX_FILE` sized?

Reply:
me not explicitly. I know some fuzzbench inputs are larger than that.
but this needs a target that actually uses that amount - otherwise it will be trimmed ...
we could test with perl/php/json targets or pcap